src/config.ts

```diff
@@ -1,21 +1,18 @@
 import { extname } from 'node:path';
 import { readFile, writeFile } from 'node:fs/promises';
 import type { CSpellSettings } from 'cspell';
+import { searchForConfig } from 'cspell-lib';
 // eslint-disable-next-line import/no-relative-packages
-import { readConfig } from '../node_modules/cspell/dist/util/fileHelper';
 import { previousState } from './shared';
 
 let configPath: string | undefined;
 
 export const findConfig = async (config?: string) => {
 	// Try to locate a config file in the current working directory, or with the `config` option if it was provided.
-	const configInfo = await readConfig(config, process.cwd());
+	const configSource = config ?? (await searchForConfig(process.cwd()))?.__importRef?.filename;
```

Comment:
`searchForConfig` might reject the promise if nothing is found. If so, you should do something along the lines of: `config ?? (await searchForConfig(process.cwd()).then((res) => res.__importRef?.filename).catch(() => null)`

Reply:
Nevermind, it doesn't, ignore me haha